action.yml

```
@@ -33,321 +33,218 @@ outputs:
-          gh pr create --base main --head "$version_branch" --title "$pr_title" --body "New Version Proposed"
-
-          echo "version_pr=1" >> $GITHUB_OUTPUT
+        source ${{ github.workspace }}/src/logging.sh
+        source ${{ github.workspace }}/src/utils.sh
+        source ${{ github.workspace }}/src/git.sh
+
+        source ${{ github.workspace }}/src/package_version_detect.sh
+        source ${{ github.workspace }}/src/package_version_update.sh
+        source ${{ github.workspace }}/src/project.sh
+
+        updated_version=$(echo "${{ steps.version_increase.outputs.updated_workspaces_info }}" | 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)